Makefile

```diff
@@ -2,24 +2,33 @@
 # https://precice.org/adapter-calculix-get-calculix.html
 # Set the following variables before building:
 # Path to original CalculiX source (e.g. $(HOME)/ccx_2.xx/src )
-CCX_VERSION			= 2.20
-CCX             = $(HOME)/CalculiX/ccx_$(CCX_VERSION)/src
+CCX_VERSION	= 2.21
+CCX_HOME        = /opt/src
+CCX             = $(CCX_HOME)/CalculiX/ccx_$(CCX_VERSION)/src
```

Comment:
The `CCX_VERSION` should indeed be updated, but the `CCX` should remain as-is.
I don't think we need a separate `CCX_HOME` variable, it makes copy-pasting paths more complicated. If it helps further parts or CalculiX-internal workflows, let's keep it.